api/app/weather_models/precip_rdps_model.py

```diff
@@ -36,6 +36,10 @@ async def compute_and_store_precip_rasters(model_run_timestamp: datetime):
         for hour in range(0, 36):
             accumulation_timestamp = model_run_timestamp + timedelta(hours=hour)
             (precip_diff_raster, geotransform, projection) = await generate_24_hour_accumulating_precip_raster(accumulation_timestamp)
+            if precip_diff_raster is None:
```

Comment:
Is it possible to include a test for this case?

Reply:
For sure! It's not very elegant but it works.
https://github.com/bcgov/wps/pull/3996/commits/4eddc12b45b06f49b12d2fc633d5f589de4fbd72